src/ophys_etl/modules/suite2p_registration/schemas.py

```diff
@@ -94,6 +94,29 @@ class Suite2PRegistrationInputSchema(argschema.ArgSchema):
         default=8,
         description="Maximum number of iterations to preform when creating a "
                     "reference image.")
+    remove_empty_frames = argschema.fields.Boolean(
```

Comment:
Is there a reason you did not expose the `n_sigma` for `find_movie_start_end_empty_frames`? I could imagine wanting to change that value in some extremely low SNR movies.

Reply:
I mostly wanted to have it be a parameter in the function to avoid "magic numbers". I don't see the utility too much in exposing it. It's already be run on low signal to noise data in the 3P experiments and does fine. I'm happy to not expose the setting for now.